### ISSUE TITLE
Outlook 2013 x64: NVDA does not read the IMAP folder list and throws an error when focussing a list item

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5442]
**Reported by bramd on 2015-10-26 22:55**
Steps to reproduce:
1. Open Outlook
2. Focus an IMAP account in the folder pane
3. Select "IMAP folders" from the context menu
4. Press the "Query" button
5. Focus the folder list

Log follows:

ERROR - eventHandler.executeEvent (23:50:30):
error executing event: gainFocus on <NVDAObjects.IAccessible.sysListView32.ListItem object at 0x05623810> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in **init**
  File "eventHandler.pyc", line 98, in next
  File "C:\Users\bramd\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient__init__.py", line 310, in event_gainFocus
  File "eventHandler.pyc", line 98, in next
  File "NVDAObjects__init__.pyc", line 857, in event_gainFocus
  File "NVDAObjects\behaviors.pyc", line 468, in reportFocus
  File "NVDAObjects__init__.pyc", line 801, in reportFocus
  File "speech.pyc", line 363, in speakObject
  File "speech.pyc", line 280, in speakObjectProperties
  File "baseObject.pyc", line 34, in **get**
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\sysListView32.pyc", line 386, in _get_name
  File "NVDAObjects\IAccessible\sysListView32.pyc", line 333, in _getColumnContent
  File "NVDAObjects\IAccessible\sysListView32.pyc", line 325, in _getColumnContentRaw
  File "winKernel.pyc", line 124, in readProcessMemory
ArgumentError: argument 2: <type 'exceptions.OverflowError'>: long int too long to convert
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (23:50:31):
Developer info for navigator object:
name: exception: argument 2: <type 'exceptions.OverflowError'>: long int too long to convert
role: ROLE_LISTITEM
states: STATE_FOCUSABLE, STATE_SELECTABLE, STATE_FOCUSED, STATE_SELECTED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.sysListView32.ListItem object at 0x05623810>
Python class mro: (<class 'NVDAObjects.IAccessible.sysListView32.ListItem'>, <class 'NVDAObjects.behaviors.RowWithFakeNavigation'>, <class 'NVDAObjects.behaviors.RowWithoutCellObjects'>, <class 'NVDAObjects.IAccessible.sysListView32.ListItemWithoutColumnSupport'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (276, 349, 347, 17)
value: None
appModule: <'outlook' (appName u'outlook', process ID 2572) at address 9cdce30>
appModule.productName: u'Microsoft Outlook'
appModule.productVersion: u'15.0.4701.1000'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 1180682
windowClassName: u'SysListView32'
windowControlID: 206
windowStyle: 1344344073
windowThreadID: 4984
windowText: u'Mailboxes'
displayText: u'[Gmail]/Alle berichten'
IAccessibleObject: <POINTER(IAccessible) ptr=0xad89948 at 51ba030>
IAccessibleChildID: 2
IAccessible event parameters: windowHandle=1180682, objectID=-4, childID=2
IAccessible accName: u'\u85ff\u75c0\u441a\u488d\u4c01\uc38b\u958b\u0c24'
IAccessible accRole: ROLE_SYSTEM_LISTITEM
IAccessible accState: STATE_SYSTEM_SELECTABLE, STATE_SYSTEM_SELECTED, STATE_SYSTEM_MULTISELECTABLE, STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (19922950)
IAccessible accDescription: exception: (-2147352573, 'Member not found.', (None, None, None, 0, None))
IAccessible accValue: None

**Blocked by #3339**

**Comment by bhavyashah** [comment]
Clearly, this ticket is probably dependent on #3339 if @jcsteh's guess in https://github.com/nvaccess/nvda/issues/5442#issuecomment-155344140 is correct. Which of the following should be done? CC @feerrenrut 
* Add the blocked label to this ticket
* Close this ticket as a duplicate
* Do nothing until #3339  is worked on

**Comment by feerrenrut** [comment]
Closing as a duplicate of #3339
